lib/ContactInfo.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import { Link } from 'react-router'
+
+export default React.createClass({
+  getDefaultProps() {
+    return {
+      cont: {
```

Comment:
Although this works, I would opt for being very explicit in the wording here. Data is extremely important, lets not leave these words up to interpretation. Perhaps `contacts` would be better.